pep-0001.txt

```diff
@@ -486,9 +486,8 @@ Each PEP should have the following parts/sections:
 
 12. References -- A collection of URLs used as references through the PEP.
 
-13. Copyright/public domain -- Each PEP must either be explicitly
-    labeled as placed in the public domain (see this PEP as an
-    example) or licensed under the `Open Publication License`_.
+13. Copyright/license -- Each PEP must be placed under a dual license of
```

Comment:
Hello @brettcannon,

What about the existing PEPs? Should them be re-licensed?
If no, I suggest to do a minor change here.

```suggestion
13. Copyright/license -- Each new PEP must be placed under a dual license of
```

Reply:
Thanks! No, there's no plan to re-license old PEPs.